doc/dev/mfx_trace2-dev.md

```diff
@@ -0,0 +1,75 @@
+# MediaSDK tracing (mfx_trace2). Developer documentation
+
+## How to use mfx_trace2?
+
+See [mfx_trace2 user documentation](./mfx_trace2.md).
+
+## Overview
+
+Main trace system object is presented by global instance of `mfx::Trace` class (instance is called `_mfx_trace`).
+Its constructor (`mfx::Trace::Trace()`) reads configuration file (on Linux) or registry values (on Windows), initializes all enabled backends.
+
+User is not supposed to touch internals of `mfx::Trace` class except `mfx::Trace::Scope` which is public interface for user to enable scoped tracing. Scope saves all events in std::vector `_mfx_trace.events` where all backends can find them using `mfx::Trace::pushEvent` function.
+
+Event saves the following information:
```

Comment:
Maybe you need to add mfx::trace::scope key API here in a same way you did below for TextLog.

Reply:
mfx::trace::scope key API is already shown in user documentation. But do we need to add mfx::Trace interface here? I think all functions that may be needed for developer are mentioned here.

Reply:
```
namespace mfx {
class Trace {
public:
  Trace();
  void pushEvent(const Trace::Event &e);
  
  struct Event { <variables> };
  class Scope {
    public:
      Scope(SourceLocation sl, const char* name, mfxU8 level = GENERIC);
        Scope(SourceLocation sl, const char* name, const char *category, mfxU8 level = GENERIC);
        ~Scope();

        template <typename... Args>
        void event(SourceLocation sl, const char *key, Args... args)
  }
};
}
```

Reply:
Added information about `mfx::Trace` interface here